enhancements/installer/aws-outposts-remote-workers.md

```diff
@@ -0,0 +1,215 @@
+
+Phase 2:
+- Admin can use ROSA cli or UI to deploy OCP/Hypershift on AWS Outpost
+
+### Non-Goals
+
+We do not intend to implement AWS Outpost specific features in OCP as well us we
+do not plan to handle long lasting networking issues between regular AWS region andan outpost.
+
+## Proposal
+
+Phase 0:
+We want to provide [documented](https://github.com/openshift/openshift-docs/pull/53265) manual procedure on how to deploy an OCP cluster in a regular
+AWS region with worker nodes running in AWS Outpost. The user needs to create VPC and
+subnets before the installation and provide them in the install-config to prevent VPC
+creation by IPI installer. The subnets need to be created both in the regular region and
+in the outpost (tagged as "kubernetes.io/cluster/<non-cluster-name>": "owned"). In the
+install-config only AWS region subnet needs to be used so NLB is created for the
+apiserver to use. During manifest stage it is needed to change workers' machineSet
+to use subnets created in the outpost. As the last modification the user needs to
+update MTU for the network provider due to AWS Outpost supporting only 1300 bytes. Once the
+cluster is deployed the user needs to install AWS load balancer operator and configure
+ALB ingress.
+
+Phase 1:
+Separate enhacement needed
+
+Phase 2:
+Separate enhacement needed
+
+### Workflow Description
+
+Phase 0:
+In order to start the cluster installation process the user needs to follow steps to
+configure AWS account and AWS Outpost instance types. Next the user needs to create
+VPC with subnets for both the AWS region and the outpost. The user needs to create
+cloud formation template with AWS outpost details and use aws client to run it.
+Next the user needs to create an install-config in the specified directory and modify it
+with AWS details according to [documentation](https://github.com/openshift/openshift-docs/pull/53265). Once it is ready the user needs to
+create manifests. MachineSet needs to be updated with AWS outpost subnet information
+and MTU. Now we can create the cluster and after the installation we need to install
+AWS load balancer operator and configure ALB based ingress.
+
+Phase 1:
+Separate enhacement needed
+
+Phase 2:
+Separate enhacement needed
+
+#### Variation [optional]
+
+N/A
+
+### API Extensions
+
+N/A - no api extensions are being introduced in this proposal
+
+### Implementation Details/Notes/Constraints [optional]
+
+NLB is not supported by AWS outpost and it is needed for apiserver to work correctly.
+That is why in Phase 0 we have decided to run master nodes in the regular AWS region and only
+workers in AWS outpost where we can configure ALB for workload ingress.
+
+### Risks and Mitigations
+
+Administrator may want to use other storage types than gp2 (supported by outpost).
+In Phase 0 we will make gp2 storageclass as default and later make sure that gp3
+storageClass is not created.
```

Comment:
https://github.com/openshift/openshift-docs/pull/53265 shows how to mark `gp3-csi` as non-default, but it does not delete it.

You can either rephrase this PR or update the docs - there will be a way to tell OCP not to create any StorageClasses in 4.13 via https://github.com/openshift/enhancements/pull/1152/files (the enhancement says 4.12, but it slipped to 4.13).

Reply:
I updated the section. I hope it is align with your comment.